accounts/manager.go

```diff
@@ -93,6 +106,16 @@ func (am *Manager) Config() *Config {
 	return am.config
 }
 
+// AddBackends starts the tracking of additional backends for wallet updates.
+// cmd/geth assumes once this func returns the backends have been already integrated.
+func (am *Manager) AddBackends(backends ...Backend) {
+	done := make(chan struct{}, len(backends))
+	for _, backend := range backends {
+		am.newBackends <- newBackendEvent{backend, done}
+		<-done
```

Comment:
We should also check if the `quit` is closed, otherwise this can be blocked forever.

Reply:
Quit is not closed ever though, it's a `chan chan error`. You'd need to add a `term chan struct{}` that gets closed if you want to check for it. Also, I think the block can only happen in `am.newBackends <-`, since once we read the event, we will surely respond.

Reply:
```go
select {
  case am.newBackends <- newBackendEvent{backend, done}:
  case <-am.term:
    return
}
```